packages/role-governance/.gitignore

```diff
@@ -1,4 +1,10 @@
-node_modules
```

Comment:
What is the reason of having git configuration in packages instead of root?

Reply:
Hmm, you are right. I would remove it.